python_ta/debug/accumulation_table.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+import inspect
+from typing import Union
+
+import astroid
+import tabulate
+
+
+class AccumulationTable:
+    """
+    Class used as a form of print debugging to analyze different loop and
+    accumulation variables during each iteration in a for loop
+
+    Private instance attributes:
+        _num_iterations: the number of for loop iterations
+        _accumulation_names: the variable names of the different accumulation variables
+        _loop_accumulation_values: a mapping between the accumulation variables and their value during each iteration
+        _loop_var_name: the name of the loop variable
+        _loop_var_val: the value of the loop variable during each iteration
+
+    """
+
+    _num_iterations: int
+    _accumulation_names: list[str]
+    _loop_accumulation_values: dict[str, list]
+    _loop_var_name: str
+    _loop_var_val: list[Union[int, str]]
+
+    def __init__(self, accumulation_names: list) -> None:
```

Comment:
specify the contained type in the `list` (and check the other type annotations in this file)

Reply:
@harshkpatel I think you missed this comment